Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.cxx

```diff
@@ -80,8 +81,8 @@ vtkMRMLMarkupsDisplayNode::vtkMRMLMarkupsDisplayNode()
   this->PropertiesLabelVisibility = true;
   this->PointLabelsVisibility = false;
 
-  this->ActiveComponentType = ComponentNone;
-  this->ActiveComponentIndex = -1;
+  // Set active component defaults for mouse (identified by empty string)
+  this->ActiveComponents[""] = ComponentInfo();
```

Comment:
What do you think of introducing a static variable called `DefaultContextName` ?

Reply:
Python wrapping fails when I want to use a const static string variable as a default value for a function parameter (like int GetActiveControlPointForContext(std::string context=DEFAULT_CONTEXT_NAME);)
Any advice?

Reply:
@dgobbi  Out of curiosity, do you know if this is expected to work in the latest VTK ? If yes, then we will backport the corresponding change into the Slicer/VTK fork  (we are still stuck with VTK of before the major build system overhaul)

Reply:
I just tested with VTK master and it seems to work as expected, but I don't remember any specific change that enabled it.  What is the error?

Reply:
The error is `vtkMRMLMarkupsDisplayNodePython.cxx(451): error C2065: 'DEFAULT_CONTEXT_NAME': undeclared identifier`
It is a static const std::string defined in the header. The problem is that in the Python cxx there is no such declaration and when it's used in the function headers it is undefined.

Reply:
Show me the file where `DEFAULT_CONTEXT_NAME` is declared and I'll figure out why it isn't visible in this context, the fact that the type is string is probably a red herring.  The wrappers are probably just not properly scoping a constant.

Reply:
Thank you, @dgobbi !
Here are the files: https://gist.github.com/cpinter/ba47dd9075a5afa5aae012b08c31e927

Reply:
The declaration `static const std::string DEFAULT_CONTEXT_NAME;` is not wrapped since the wrappers only handle POD constants (no objects).  But I can modify the wrappers to handle the default value in cases like this.

Reply:
@dgobbi Thanks for looking into this :pray:  supporting this would be neat.

@cpinter In the mean time, we could also add a static function `GetDefaultContextName()` like what is done in [vtkMRMLSubjectHierarchyConstants](https://github.com/Slicer/Slicer/blob/master/Libs/MRML/Core/vtkMRMLSubjectHierarchyConstants.h)

Reply:
Added in https://github.com/Slicer/Slicer/commit/110e94795d768d772f6ff10536b1521e7a50e63f